packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1600,8 +1602,12 @@ class Padding extends SingleChildRenderObjectWidget {
 ///   of [FlutterLogo]/2 + height of [FlutterLogo]/2) = (36.0, 48.0).
 ///
 /// The [Alignment] class uses a coordinate system with an origin in the center
-/// of the [Container], and so [Align] will place the [FlutterLogo] at (36.0, 48.0)
-/// according to this system.
+/// of the [Container], as shown with the [Icon] above. [Align] will place the
+/// [FlutterLogo] at (36.0, 48.0) according to this coordinate system.
+///
+/// ![A blue square container with the Flutter logo positioned according to the
+/// following Alignment. A point is marked at the center of the container for the
```

Comment:
nit: is "following" still correct? Or do you mean to reference the previous paragraph?